app/routes/tasks.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+from flask import Blueprint, jsonify, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from app.helpers import *
```

Comment:
`import *` is generally discouraged since we can end up importing unexpected symbols. It's preferred to explicitly list the symbols we are importing from the module.